common/tests/values/global.yaml

```diff
@@ -0,0 +1,56 @@
+test: true
+
+global:
+  ## Configure pull secrets for all deployments
+  registryPullSecrets: []
+
+  # Use Yaml formatted string to add extra environment properties to all deployments, i.e.
+  extraEnv: |
+    # - name: SERVER_USEFORWARDHEADERS

```

Comment:
Not sure it helps to have these commented-out entries here. I think it would be clearer to remove all the comments for the purpose of the test.

Reply:
Actually maybe it does help.